packages/react-server-cli/src/parseCliArgs.js

```diff
@@ -22,6 +22,10 @@ export default (args = process.argv) => {
 			describe: "Port to start listening for react-server's JavaScript. Default is 3001.",
 			type: "number",
 		})
+		.option("bind", {
+			describe: "IP address to bind to. Default: 0.0.0.0 (any)",
+			type: "string",
+		})
```

Comment:
If we call it `bindIp` internally to disambiguate, maybe the CLI option should be `--bind-ip`, too?